### PR TITLE
Updated code to handle invalid tracking_ids for StartTracking service

### PR DESCRIPTION
**Pull request by reaganlo**
Changes proposed in this pull request:
- Updated date in copyright headers
- Updated README with information on coordinates and units
- Updated README bugs
- Refactored get_tracking_ids service to get_tracking_state
- Updated tracking_id to int32_t based on Beta3 API changes
- Removed person_detection_count as it is redundant
- Added more comments in the msg and srv descriptions
- Renamed test files.

### CODE REVIEW

File: src/person_nodelet.cpp [hunk 1]
```diff
@@ -817,31 +818,48 @@ namespace realsense_person
...
         auto single_person_data =
             person_data->QueryPersonData(PersonModule::PersonTrackingData::ACCESS_ORDER_BY_INDEX, i);
         auto detection_data = single_person_data->QueryTracking();
         auto tracking_id = detection_data->QueryId();
-        res.tracking_ids.push_back(tracking_id);
+        res.detected_tracking_ids.push_back(tracking_id);
+        if (tracking_id == tracking_id_)
+        {
+          tracking_id_found = true;
+          break;
```
Comment (kevincwells) [inline src/person_nodelet.cpp]: Why is it breaking here?  Won't this have the effect of making it stop adding detected tracking IDs once it encounters the currently tracked id?  Don't you want the full list of detected IDs at all times?

File: src/person_nodelet.cpp [hunk 2]
```diff
@@ -862,11 +880,40 @@ namespace realsense_person
...
+      {
+        auto single_person_data =
+            person_data->QueryPersonData(PersonModule::PersonTrackingData::ACCESS_ORDER_BY_INDEX, i);
+        auto detection_data = single_person_data->QueryTracking();
+        auto tracking_id = detection_data->QueryId();
+        if (tracking_id == req.tracking_id)
+        {
+          tracking_id_found = true;
+          break;
+        }
```
Comment (kevincwells) [inline src/person_nodelet.cpp]: Ah, I suspect the break above is due to a copy/paste of this code.

File: README.md [hunk 3]
```diff
@@ -126,6 +127,9 @@ Install the package and its dependent packages as follows:
 
    Stops tracking and disables the tracking feature in the middleware.
 
+###Custom Messages
+Refer to the defintion at ([Custom Messages](msg/))
+
```
Comment (reaganlo) [inline README.md]: It is complete. Unlike topics and services, messages are technically not user facing APIs. So did not want to mention their definitions explicitly in the README. I will added a "." at the end.

File: src/person_nodelet.cpp (hunk 2 above)
Comment (reaganlo) [inline src/person_nodelet.cpp]: @kevincwells You are absolutely right. Great catch. Thanks for reviewing this so fast.

**Pull request merged**